### PR TITLE
Order admin

### PR DESCRIPTION
**Pull request by pipinamngua**

### CODE REVIEW

**Comment by pipinamngua** [comment]
Order admin , order detail

**Comment by pipinamngua** [comment]
ready

**Comment by pipinamngua** [comment]
Order admin , order detail

**Comment by pipinamngua** [comment]
ready

File: app/Http/Controllers/Admin/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        $order->delete();
+
+        Session::flash('success', trans('custom.order.deletesuccess'));
+        
+        return redirect()->route('order');
+    }
+
+    public function changeStatus($status, $id)
+    {
+        $order = Order::findOrFail($id);
```
Comment (trinhductoan) [inline app/Http/Controllers/Admin/OrderController.php]: viết sang scope đi em

Comment (pipinamngua) [reply app/Http/Controllers/Admin/OrderController.php]: ok a

File: database/migrations/2018_05_17_033345_add_arrtrbute_into_orders_table.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class RenameNewsIntoPostsTable extends Migration
+class AddArrtrbuteIntoOrdersTable extends Migration
```
Comment (trinhductoan) [inline database/migrations/2018_05_17_033345_add_arrtrbute_into_orders_table.php]: không sửa file migrate, tạo file mới em nhé

**Comment by trinhductoan** [comment]
commented

**Comment by pipinamngua** [comment]
write scope in OrderDetail

**Comment by pipinamngua** [comment]
ready

File: app/Models/Order.php [hunk 3]
```diff
@@ -22,4 +22,19 @@ public function products()
     {
         return $this->belongsToMany(Product::class, 'order_details', 'order_id', 'product_id');
     }
+
+    public function order_details()
```
Comment (trinhductoan) [inline app/Models/Order.php]: tên hàm viết sai conventions
`orderDetails`

File: app/Models/Order.php [hunk 4]
```diff
@@ -22,4 +22,19 @@ public function products()
...
+    {
+        return $this->hasMany(OrderDetail::class);
+    }
+
+    public function user()
+    {
+        return $this->belongsTo(User::class);
+    }
+
+    public function scopeFindId($query, $id)
```
Comment (trinhductoan) [inline app/Models/Order.php]: scope này có cần thiết không?

File: resources/assets/css/admin/admin.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#informUser {
+	height: 300px;
+}
+
+.avatar-user {
+	height: 200px;
```
Comment (trinhductoan) [inline resources/assets/css/admin/admin.css]: indent

**Comment by trinhductoan** [comment]
commented

**Comment by pipinamngua** [comment]
edit

**Comment by pipinamngua** [comment]
ready

**Comment by pipinamngua** [comment]
edit scope in Order model

**Comment by pipinamngua** [comment]
ready

**Comment by pipinamngua** [comment]
edit scope in Order model

**Comment by pipinamngua** [comment]
ready

**Pull request merged**